tasks/InfracostSetup/task.json

```diff
@@ -9,7 +9,7 @@
   "author": "Infracost",
   "version": {
     "Major": 0,
-    "Minor": 1,
+    "Minor": 2,
     "Patch": 18
```

Comment:
```suggestion
    "Patch": 0
```

Reply:
done in https://github.com/infracost/infracost-azure-devops/pull/45/commits/51e7994d795d728acc2fcd1bef0007f8f0344df2